src/Button/Button.styles.ts

```diff
@@ -0,0 +1,45 @@
+import styled, { css } from 'styled-components';
+import { Typography } from '../Typography';
+import { SIZES_MAPPING } from './constants';
+import { ButtonVariants, ColorTypes, SizeTypes } from './types';
+import { getButtonVariantStyles, getTypographyVariantStyles } from './utils';
+
+interface StyledButtonProps {
+  $color: ColorTypes;
+  $size: SizeTypes;
+  $variant: ButtonVariants;
+  disabled: boolean;
+}
+
+interface StyledTypographyProps {
+  $color: ColorTypes;
+  $variant: ButtonVariants;
+  disabled: boolean;
+}
+
+const BaseButtonStyles = css<StyledButtonProps>`
+  ${({ theme, $color, $variant, disabled }) => getButtonVariantStyles(theme, $color, $variant, disabled)}
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  height: ${({ $size }) => SIZES_MAPPING[$size]};
+  minimum-width: 80px;
```

Comment:
```suggestion
  min-width: 80px;
```
Was this minimum width specified somewhere?